### PR TITLE
[9.0][partner_firstname] Update fr.po

### PR DESCRIPTION
**Pull request by Ellyan**
Adding translation for :
"First name"
"Last name"
"No name is set."

### CODE REVIEW

**Comment by dreispt** [comment]
:+1:

File: partner_firstname/i18n/fr.po [hunk 1]
```diff
@@ -36,13 +36,13 @@ msgstr "Est une société?"
...
 msgid "Last name"
-msgstr ""
+msgstr "Nom de famille"
 
 #. module: partner_firstname
 #: code:addons/partner_firstname/exceptions.py:23
 #, python-format
 msgid "No name is set."
-msgstr ""
+msgstr "Pas de nom défini"
```
Comment (yvaucher) [inline partner_firstname/i18n/fr.po]: To translate it exactly it misses the dot at the end of sentence or is there a reason to remove it?

Comment (Ellyan) [inline partner_firstname/i18n/fr.po]: No, I did not see the dot at the time.

**Comment by rafaelbn** [comment]
Hi @Ellyan please could recolve conflics to check if we can merge this one? thanks

**Comment by yvaucher** [comment]
Terms are already synchronised with Transifex:

https://github.com/OCA/partner-contact/blob/9.0/partner_firstname/i18n/fr.po

We can close this PR

**Comment by yvaucher** [comment]
Thanks for the effort nevertheless

**Pull request closed without merging**